### DISCUSSION TITLE
Discord callback keeps giving 401 regardless

### DISCUSSION THREAD

**Discussion by callumok2004** [nextauthjs/next-auth#3065] [Help]
![image](https://user-images.githubusercontent.com/35779365/139435390-b5b3dc86-6e43-4b59-b3af-fe8986593da0.png)

**Comment by callumok2004** [comment]
Any ideas?

**Reply by ItsAzaria** [reply]
works for me, make sure your token + id are correct and being picked up

**Reply by callumok2004** [reply]
Everything is correct, it just refuses to work.
